docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
 
-  The table below provides example entries for large scale food fortification with iron.
 
-.. list-table:: Affected Outcomes
+Please see this memo for a summary of the studies and the effect sizes :download:`SQ-LNS interventions memo<sqlns_memo_das2019.docx>`
+  
+.. list-table:: Affected Outcomes by intervention
   :widths: 15 15 15 15
   :header-rows: 1
 
   * - Outcome
     - Effect
     - Modeled?
     - Note (ex: is this relationship direct or mediated?)
-  * - Hemoglobin concentration
-    - Increases population mean
+  * - Moderate wasting
+    - Reduces prevalence
     - Yes
-    - 
-  * - Malaria
-    - Increases incidence rate
+    - Direct
+  * - Moderate stunting
+    - Reduces prevalence
+    - Yes
+    - Direct
+  * - Severe stunting
+    - Reduces prevalence
+    - Yes
+    - Direct
+  * - Moderate underweight
+    - Reduces prevalence
     - No
-    - 
+    - Direct
+  * - Anaemia
+    - Reduces prevalence
+    - No
+    - Direct
+
 
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
-.. todo::
-
-  Document known baseline coverage data, using the table below if appropriate
+No baseline coverage of SQ-LNS
 
-.. list-table:: Baseline coverage data
-  :widths: 15 15 15 15 15
-  :header-rows: 1
-
-  * - Location
-    - Subpopulation
-    - Coverage parameter
-    - Value
-    - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
 
 Vivarium Modeling Strategy
 --------------------------
 
-.. todo::
+:underline:`Outcome moderate wasting`
+
+For the outcome moderate wasting, the review compared prevalence of **moderate wasting** at 18 or 24 months between intervention and control children. LNS plus complementary feeding reduced the prevalence of moderate wasting by 18% (RR 0.82, 95% CI 0.74 to 0.91; eight studies; 13,172 participants; moderate-quality evidence). There was *no impact* of LNS plus complementary feeding on **severe wasting** (RR 1.27, 95% CI 0.66 to 2.46; three studies, 2329 participants)
+
+We will apply the relative risk ratio as a relative rate ratio on the incidence of MAM from MILD (i2) starting from the age-start of the intervention starts (6 months). This is because we assume that the intervention does not affect the duration of disease and hence:
 
-  Add an overview of the Vivarium modeling section.
+| Prevalence_intervention 
+| = 0.82 x (prevalence_baseline_6mo) 
+| = 0.82 x (incidence_baseline_6mo) x Duration_baseline
 
 .. todo::
+  
+  Think about whether we want to apply the full reduction in incidence at 6 months or we have a gradual reduction in incidence, reaching full reduction at 18 months? 
```

Comment:
Think about whether we need to apply a reduction to birth prevalence, as well, despite the illogic of an intervention at age 6 months affecting a risk exposure at 0 months.

Reply:
My thinking is that there is no reduction until 6 months. So 0-6 month olds stay the same as baseline.

Reply:
What are you thinking here, @aflaxman? Is the concern that somehow our prevalences might get thrown off again if we don't also alter birth prevalence?

Reply:
> What are you thinking here, @aflaxman? Is the concern that somehow our prevalences might get thrown off again if we don't also alter birth prevalence?

Exactly my concern.  But let's try it the logical way, and revisit if it does not look well-calibrated.

Reply:
We don't expect prevalence to match GBD in this scenario. We are actively trying to decrease prevalence with a shorter duration (and incidence remain the same as baseline).

Reply:
Hence prevalence of 0-6 month olds will be the same as baseline. Once the SQ-LNS intervention starts, the incidence of i2 decreases and we should see a decrease in prevalence among 6mo+.

Reply:
We also expect 0% coverage of SQ-LNS at baseline.

Reply:
@aflaxman @NathanielBlairStahn